src/ui/store/slice/shareData/test/shareDataSlice.spec.ts

```diff
@@ -122,6 +122,22 @@ describe('Given the share data slice', () => {
     value: O.some(['Agriculture', 'Open Data', 'Dataviz'])
   }
 
+  const dateStringRangeItemID6: FormItem = {
+    id: '6',
+    title: 'date range',
+    required: false,
+    type: 'date-range',
+    value: O.some({ from: '2023-09-21', to: '2023-09-30' })
+  }
+
+  const emptyDateStringRangeItemID6: FormItem = {
```

Comment:
Could be good to add a case when just one property is provided and the other is `null`

Reply:
Yes. Test added! Thanks